.github/workflows/build-nginx-on-release.yml

```diff
@@ -1,20 +1,23 @@
-name: Build Containers
+name: Build Containers On Chart Release
```

Comment:
We want both build on push and build on release, otherwise we can't test changes until they're released